src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -76,17 +77,20 @@ def all(dict_values):
     # Perform basic (limited) check for moduel completeness
     C1.check_for_sufficient_assets_on_busses(dict_values)
 
+    # just to be safe, run evaluation a second time
+    C1.check_for_label_duplicates(dict_values)
```

Comment:
Why not running it only once, ie remove execution on line 56?

Reply:
There are some Assets that are created automatically - I wanted to catch them with the second execution as well. Not sure if only the second one would be sufficient...

Reply:
I will leave this for now as is, as I do not have enough brain capacity to decide this now. Also, it does not have a negative effect on the results (only might take a bit longer)